lightwood/model/neural.py

```diff
@@ -261,18 +263,12 @@ def fit(self, ds_arr: List[EncodedDs]) -> None:
         criterion = self._select_criterion()
         scaler = GradScaler()
 
-        for subset_itt in (0, 1):
-            for subset_idx in range(len(dev_ds_arr)):
-                train_dl = DataLoader(
-                    ConcatedEncodedDs(train_ds_arr[subset_idx * 9: (subset_idx + 1) * 9]),
-                    batch_size=200, shuffle=True)
+        train_dl = DataLoader(ConcatedEncodedDs(train_ds_arr), batch_size=200, shuffle=True)
```

Comment:
I was suggesting removing the "warmup" subset iteration, I doubt removing it entirely will help, given that I tried that at some point and the results were much worst. But training methodology has changed a bit since then, so who knows. Let's look at the benchmarks.

Reply:
An additional point - batch_size should be a passable parameter